src/engine/engine_common.c

```diff
@@ -599,6 +599,8 @@ static int _ocf_engine_refresh(struct ocf_request *req)
 {
 	int result;
 
+	ocf_debug_request_trace(req, ocf_req_refresh, 0);
+
```

Comment:
Having the tracking on the entry point to an engine loses information about the original source of the request. For example, there are three places where req can be redirected from engine_rd to engine_pt, but tracking only the entry point we don't know exactly which path has been taken.

The better solution would be to have this tracking on the redirect point. So for all the request the first redirect point would be `ocf_engine_hndl_req()` or `ocf_engine_hndl_fast_req()` (and then we automatically know if it's fast path or not, even if it's not an engine_fast!). Then we need to find all the other redirect points within the engines, and call the tracing function from there.